pcsx2/VUmicro.cpp

```diff
@@ -20,6 +20,19 @@
 #include "GS.h"
 #include "Gif_Unit.h"
 
+__inline u32 CalculateMinRunCycles(u32 cycles)
+{
+	// If we're running an interlocked COP2 operation
+	// run for an exact amount of cycles
+	if((cpuRegs.code & 0xFC000001) == 0x48000001)
```

Comment:
Is `code` going to be correct in all cases here? Iirc VCALLMS is a fallback but are there pathways here from other instructions which won't be? Might need a global flag for "in interlocked op"..

Reply:
We always use cpuRegs.code to check if it's interlocking if you check in microVU_Macro.inl and I crafted it specifically to ignore the "V" instructions because for those to run it has to finish the VU program that's running, plus it avoids false positives, so this cos is kind of irrelevant and doesn't matter if it runs a bit longer :)

Reply:
Right, but this routine gets called when the block is executing, not compiling right?

Reply:
tits, uhh you make a good point...

Reply:
Alright, I've changed it up so if the EE block uses interlocking, the rest of the sync points on the block will also use accurate cycles, so at runtime it should be correct. I always mix up compile time and run time lol